src/clang.rs

```diff
@@ -387,16 +387,26 @@ impl Cursor {
 
     /// Given that this cursor's referent is a function, return cursors to its
     /// parameters.
-    pub fn args(&self) -> Vec<Cursor> {
+    pub fn args(&self) -> Option<Vec<Cursor>> {
+        // XXX: We might want to use and keep num_args
```

Comment:
We use an iterator pattern somewhere else (for comment children and attributes), so something like that can also work, and it'd be better IMO.

If you want to give a crack at it, that'd be awesome.